docs/document/docs/components/desktop/tabs.md

```diff
@@ -0,0 +1,100 @@
+---
+id: tabs
+title: Tabs
+sidebar_label: Tabs
+---
+
+## Overview
+
+A multiple tabs component that can switch displaying contents with buttons.
+
+<div class="sample-container" id="tabs">
+  <div id="sample-container__components"></div>
+</div>
+<script src="/js/samples/desktop/tabs.js"></script>
+
+---
+
+## Specification
+
+### Property
+Here is a list of properties that can be used for modifying the component:
+
+| Name   | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| className | string | ""  | Component class name | |
+| id | string | ""  | Component id name | |
+| value | string | ""  | Component id name | The first tab will be displayed if the `value` is unspecified or there is no matching value in items. |
+| borderVisible | boolean | true  | Show/Hide the border surrounding the content | |
+| visible | boolean | true | Show/Hide the component | |
+| items | Array\<Item> | [] | List of items for tab will be displayed | If the items is not an array, the error message 'items' property is not array. will throw in window console. |
+| Item.label | string | "" | Tab name | |
+| Item.value | string | "" | Tab key | `Item.value` is unique and required.<br><li>If `Item.value` is duplicated, the error message 'value' property is not unique in items. will throw in window console.<li>If `Item.value` is NOT set, the error message 'value' property is not specified in items. will throw in window console. |
```

Comment:
There is no have end close tag for <li>, please add `</li>`

Reply:
Please update remark for `Item.value` to:
Will result an error if the `Item.value` is duplicated in columns or not specified.

I refer `Column.field` from Table component: Will result an error if the field is duplicated in columns or not specified